hack/tools/release/weekly/main.go

```diff
@@ -21,14 +21,17 @@ limitations under the License.
 package main
 
 import (
+	"context"
 	"flag"
 	"fmt"
 	"os"
-	"os/exec"
 	"regexp"
+	"strconv"
 	"strings"
 	"time"
 
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
 	release "sigs.k8s.io/cluster-api/hack/tools/release/internal"
```

Comment:
```suggestion
	"github.com/google/go-github/github"
	"golang.org/x/oauth2"
	
	release "sigs.k8s.io/cluster-api/hack/tools/release/internal"
```

@typeid This will fix the lint issue.

Reply:
Interesting, I was not able to find this linting issue on my side and didn't pick that up from the lint job either. Can you share how you figured this out?